src/pay/batch_payer.py

```diff
@@ -13,7 +13,7 @@
 
 logger = main_logger
 
-MAX_TX_PER_BLOCK_tz = 200
+MAX_TX_PER_BLOCK_TZ = 400
```

Comment:
Did you get the chance to test the new values in Mainnet? Were the transactions included successfully?

Reply:
StakeNow - https://tzstats.com/oorv3ata9o44yU5UD8Uk6XtaiY639JtC85RFwqHB9WjteoJioSr/

Actually, the number could be increased even more (1000). We could double it without risks because:

```
max_operation_gas = 1040000
tz_to_tz_gas = 1027
max_tx_operations = max_operation_gas / tz_to_tz_gas = 1012
```

Reply:
I would leave some safety margin. 600 would be a good batch size in my opinion

Reply:
Changed to 600